server/src/main/java/com/testsigma/controller/TestCasesController.java

```diff
@@ -158,7 +158,7 @@ public ResponseEntity<String> bulkMarkAsDelete(@RequestBody(required = false) Ma
         }
       }
     }
-    testCaseService.markAsDelete(validIds);
```

Comment:
@nixonakash01 This added space before the line is not required. Remove this

Reply:
done